app/delegate-server/Types/ServiceError.purs

```diff
@@ -35,6 +36,9 @@ derive instance Newtype AffjaxError _
 instance Show AffjaxError where
   show = Affjax.printError <<< unwrap
 
+instance Eq AffjaxError where
+  eq = eq `on` show
```

Comment:
This looks a tad suspicious.

Reply:
Fully agree. Removed this instance in this commit https://github.com/mlabs-haskell/hydra-auction-offchain/pull/41/commits/7903a9a8ba8db04597831c78772fbd63fe397969.